vkontakte/content/metadata.json

```diff
@@ -1,11 +1,11 @@
 {
-    "name": "VKontakte",
+    "name": "VK.com",
     "pluginName": "vkontakte",
-    "author": "Krzysztof Klinikowski",
-    "email": "kkszysiu@gmail.com",
-    "version": "0.1",
+    "author": "Anton Romanov",
+    "email": "theli.ua@gmail.com",
+    "version": "0.0.1",
```

Comment:
This is a bit problematic since we're decreasing the version here. Should rather bump to 0.2, I guess?